tutorials/simple/openshift.html.haml

```diff
@@ -0,0 +1,28 @@
+---
+layout: tutorial
+title: OpenShift/Kubernetes tutorial
+---
+%script(src="/javascripts/infinispan.js")
+.row
+  %ul.breadcrumb
+    %li
+      %a{:href => "/tutorials"} Tutorials
+    %li.active OpenShift/Kubernetes
+
+.row
+  %h2 OpenShift/Kubernetes
+  %p
+  .alert.alert-info
+    This tutorial requires running cluster of OpenShift or Kubernetes. For more information look into
```

Comment:
"This tutorial needs to be executed in either OpenShift or Kubernetes. For more information refer to"